public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/SettingsSummary.tsx

```diff
@@ -0,0 +1,57 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, TimeRange } from '@grafana/data';
+import { Spinner, TimeRangeLabel, useStyles2 } from '@grafana/ui';
+
+export interface Props {
+  timeRange: TimeRange;
+  className?: string;
+  isDataLoading?: boolean;
+  timeSelectionEnabled?: boolean;
+  annotationsEnabled?: boolean;
+}
+
+export function SettingsSummary({
+  className,
+  isDataLoading = false,
+  timeRange,
+  timeSelectionEnabled,
+  annotationsEnabled,
+}: Props) {
+  const styles = useStyles2(getStyles);
+
+  return isDataLoading ? (
+    <div className={(styles.summaryWrapper, className)}>
+      <Spinner className={styles.summary} inline={true} size={14} />
+    </div>
+  ) : (
+    <div className={(styles.summaryWrapper, className)}>
+      <div className={styles.summary}>
+        {'Time range = '}
+        <TimeRangeLabel className={styles.timeRange} value={timeRange} />
+      </div>
+      <div className={styles.summary}>{`Time range picker = ${timeSelectionEnabled}`}</div>
+      <div className={styles.summary}>{`Annotations = ${annotationsEnabled}`}</div>
```

Comment:
both Time range picker and Annotations values not seem to be boolean values in the figma design.
Time range picker -> disabled/ .... enabled? there's no design when enabled
Annotations -> hide/ ... show? there's no design when showed

Reply:
I think this also should be a <p> or <span> tag, right?

Reply:
Also it wouldn't make a difference because I'm relying on the flex arrangement of them items, since it works best when resizing the screen

Reply:
Good catch on the show/hide enable/disable UI! Fixed 🙌